sdk/src/main/java/co/omise/android/ui/AuthorizingPaymentActivity.kt

```diff
@@ -265,8 +265,12 @@ class AuthorizingPaymentActivity : AppCompatActivity() {
         val resultIntent = Intent().apply {
```

Comment:
On the above line, why `private fun finishActivityWithFailure(throwable: Throwable)` and not something like `private fun finishActivityWithFailure(e: Throwable)`